db/adam-moses.csv

```diff
@@ -4,7 +4,7 @@ Seth,-3895.25,-2983.25,
 Enosh,-3790.25,-2885.25,year = int(date -1)
 Kenan,-3700.25,-2790.25,month = 13 + int((date - int(data)) * 12)
 Mahalalel,-3630.25,-2735.25,day = int((month - int(month))*30) + 1
-Jared,-3565.25,-2735.25,
+Jared,-3565.25,-2603.25,
```

Comment:
Interestingly no-one notice this bug. Wasn't it surprising that in the old graph Jared died the same year as his father Mahalalel? But there was no disaster - he had 132 more years to go!